packages/playwright/src/worker/testInfo.ts

```diff
@@ -621,15 +617,15 @@ export class TestStepInfoImpl implements TestStepInfo {
     this._attachToStep(await normalizeAndSaveAttachment(this._testInfo.outputPath(), name, options));
   }
 
-  skip(...args: unknown[]) {
+  skip = wrapFunctionWithLocation((location: Location, ...args: unknown[]) => {
```

Comment:
Can we assign in the constructor, like we do in `TestInfoImpl`, instead of assigning in the class definition?